<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArsenalWindow</class>
 <widget class="QWidget" name="ArsenalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>512</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="waitingLabel">
   <property name="geometry">
    <rect>
     <x>-8</x>
     <y>-1</y>
     <width>1111</width>
     <height>751</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 65pt;
color: rgb(235, 0, 0);
border: 5px solid rgb(235, 0, 0);
</string>
   </property>
   <property name="text">
    <string>ARSENAL</string>
   </property>
  </widget>
  <widget class="QLabel" name="bombArsenal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="missileArsenal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>179</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="radarArsenal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>367</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gunArsenal">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>300</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="shieldArsenal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>245</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>610</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Waiting</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
